moveit_planners/test_configs/prbt_ikfast_manipulator_plugin/CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ install(
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
   RUNTIME DESTINATION bin
-  INCLUDES DESTINATION include
+  INCLUDES DESTINATION include/prbt_ikfast_manipulator_plugin
```

Comment:
I belive this should be `moveit_resources_prbt_ikfast_manipulator_plugin` as that is the name of the project.